Source/Core/Core/Boot/Boot.cpp

```diff
@@ -351,6 +351,44 @@ bool CBoot::Load_BS2(const std::string& boot_rom_filename)
   // Run the descrambler over the encrypted section containing BS1/BS2
   ExpansionInterface::CEXIIPL::Descrambler((u8*)data.data() + 0x100, 0x1AFE00);
 
+  // Perform a surgical modification on known IPLs for Emulated Memory Size Override.
+  if (Config::Get(Config::MAIN_RAM_OVERRIDE_ENABLE))
```

Comment:
This block should go in its own function so all of the memory override stuff is self contained.